www/js/survey/multilabel/MultiLabelButtonGroup.tsx

```diff
@@ -100,11 +100,13 @@ const MultilabelButtonGroup = ({ trip, buttonsInline=false }) => {
           )
         })}
       </View>
-      <View>
-        <IconButton icon='check-bold' mode='outlined' size={16} onPress={verifyTrip}
-                    disabled={trip.verifiability != 'can-verify'}
-                    style={{width: 20, height: 20, margin: 3}}/>
-      </View>
+      {trip.verifiability === 'can-verify' && (
```

Comment:
I see that this logic is unmodified, so I am going to merge it. But I would like to understand what it does, maybe illustrated with a short video.

From the code, it looks like if either button is yellow, we can "confirm" it. So if I had one yellow label and one blank label, I could press the confirm button and it would turn the yellow -> blue but leave the blank label unchanged.

Is that correct?

Reply:
My apologies for the late reply! Here are two videos, comparing the old UI and the changes this added.

https://github.com/e-mission/e-mission-phone/assets/98350084/373773f5-4c55-43ac-94ad-0b225e26ba5a


https://github.com/e-mission/e-mission-phone/assets/98350084/952bfc8a-1e81-4dff-8975-eba55e8bf4a2